articles/batch/batch-sig-images.md

```diff
@@ -125,6 +125,71 @@ private static void CreateBatchPool(BatchClient batchClient, VirtualMachineConfi
 }
 ```
 
+## Create a pool from a Shared Image using Python
+
+You can also create a pool from a Shared Image using Python SDK. 
+
+```python
+# Import the required modules from the
+# Azure Batch Client Library for Python
+import azure.batch._batch_service_client as batch
+import azure.batch.models as batchmodels
+from azure.common.credentials import ServicePrincipalCredentials
+
+# Specify Batch account and service principal account credentials
+account = "{batch-account-name}"
+batch_url = "{batch-account-url}"
+ad_client_id = "{sp-client-id}"
+ad_tenant = "{tenant-id}"
+ad_secret = "{sp-secret}"
+
+# Pool settings
+pool_id = "LinuxNodesSamplePoolPython"
+vm_size = "STANDARD_D2_V3"
+node_count = 1
+
+# Initialize the Batch client with Azure AD authentication
+creds = ServicePrincipalCredentials(
+    client_id=ad_client_id,
+    secret=ad_secret,
+    tenant=ad_tenant,
+    resource="https://batch.core.windows.net/"
+)
+config = batch.BatchServiceClientConfiguration(creds, batch_url)
+client = batch.BatchServiceClient(creds, batch_url)
+
+# Create the unbound pool
+new_pool = batchmodels.PoolAddParameter(id=pool_id, vm_size=vm_size)
+new_pool.target_dedicated = node_count
+
+# Configure the start task for the pool
+start_task = batchmodels.StartTask(
+    command_line="printenv AZ_BATCH_NODE_STARTUP_DIR"
+)
+start_task.run_elevated = True
+new_pool.start_task = start_task
```

Comment:
nit: add this to the init

Reply:
Thanks for the comments. I've updated the code to accommodate all suggestions. Thanks.